internal/state/rollback.go

```diff
@@ -65,10 +65,11 @@ func Rollback(bs BlockStore, ss Store, removeBlock bool) (int64, []byte, error)
 		return -1, nil, err
 	}
 
+	nextHeight := rollbackHeight + 1
 	valChangeHeight := invalidState.LastHeightValidatorsChanged
 	// this can only happen if the validator set changed since the last block
```

Comment:
Do you mind updating the comment here. Mentioning, in short, how we define `valChangeHeight`? You may also add a link to the existing issue.